Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -9,7 +9,8 @@
 import SwiftLintFramework
 import XCTest
 
-// swiftlint:disable:next type_body_length
+// swiftlint:disable type_body_length
```

Comment:
make sure you're using SwiftLint 0.13.2, since support for this syntax was only recently added